src/web/app/components/sessions-table/submission-status-tooltip.pipe.ts

```diff
@@ -13,14 +13,17 @@ export class SubmissionStatusTooltipPipe implements PipeTransform {
    * Transforms {@link FeedbackSessionSubmissionStatus} to a tooltip description.
    */
   transform(status: FeedbackSessionSubmissionStatus): string {
-    let msg: string = 'The feedback session has been created';
+    let msg: string = 'The feedback session';
 
     switch (status) {
       case FeedbackSessionSubmissionStatus.VISIBLE_NOT_OPEN:
       case FeedbackSessionSubmissionStatus.OPEN:
       case FeedbackSessionSubmissionStatus.GRACE_PERIOD:
       case FeedbackSessionSubmissionStatus.CLOSED:
-        msg += ', is visible';
+        msg += ' is visible';
+        break;
+      case FeedbackSessionSubmissionStatus.NOT_VISIBLE:
+        msg += ' is not visible to students'; // review needed as this case isn't covered in the issue.
```

Comment:
@damithc Are you good with this?
![image](https://user-images.githubusercontent.com/32454748/123646953-41f28080-d85a-11eb-905a-2353140b332c.png)

Reply:
> After:
> ![image](https://user-images.githubusercontent.com/32454748/123647399-9e55a000-d85a-11eb-85c1-305cf86ce104.png)

@pPris How about `The feedback session is not yet open for submissions` ?

Reply:
@damithc 
I've changed it accordingly, this is how it looks like now (NOT_VISIBLE status)
![image](https://user-images.githubusercontent.com/49683462/123769938-63a44400-d8fc-11eb-9ba3-7f1d41cdc877.png)



For comparison, this is how the VISIBLE_NOT_OPEN tooltip looks like
![image](https://user-images.githubusercontent.com/49683462/123770054-83d40300-d8fc-11eb-818f-d12258645aa6.png)


Does this look okay?

Reply:
Sorry @pPris , some further tweaks (I forgot that we should also mention the visibility states):
`The feedback session is waiting to open for submissions and not yet visible to students`
`The feedback session is waiting to open for submissions but is visible to students`
Would that work?

Reply:
I've updated accordingly (the existing code mentions visibility first so i stuck to that)

Here's the current look:

NOT_VISIBLE:
![image](https://user-images.githubusercontent.com/49683462/123777889-327b4200-d903-11eb-95d6-41c11f394de6.png)

VISIBLE_NOT_OPEN:
![image](https://user-images.githubusercontent.com/49683462/123777917-38712300-d903-11eb-9072-7cc1a7a4a5bd.png)

Reply:
I'd put 'waiting to open' first as that is the more important part (awaiting sessions are almost always not visible, so that part is less important). But the current version is fine too.

Reply:
Changed to the more important part first, hope it is clearer now

![image](https://user-images.githubusercontent.com/49683462/123840154-95d79500-d940-11eb-93b2-374b3803a04c.png)

![image](https://user-images.githubusercontent.com/49683462/123840321-c28bac80-d940-11eb-9dd7-7eb26d609f28.png)

![image](https://user-images.githubusercontent.com/49683462/123840191-a0922a00-d940-11eb-8430-29f99fd8b135.png)

![image](https://user-images.githubusercontent.com/49683462/123840223-a982fb80-d940-11eb-943d-20e6c10731da.png)

Reply:
@damithc Hi prof, could i check if this is good to go?

Reply:
Looks good @pPris 👍

Reply:
Oh, should we use `respondents` instead of `students`? Technically, a session can even be targeted at instructors only.

Reply:
![image](https://user-images.githubusercontent.com/49683462/124351709-9026cb80-dc2e-11eb-99bd-58fd32558a5e.png)

![image](https://user-images.githubusercontent.com/49683462/124351710-91f08f00-dc2e-11eb-9110-9ca9859fc24d.png)

@damithc Updated all `students` both the published and submission tooltips to `respondents`

Reply:
Good. Thanks @pPris